schema/meter_provider.json

```diff
@@ -75,6 +87,27 @@
                 }
             }
         },
+        "MetricProducer": {
+            "type": "object",
+            "additionalProperties": true,
+            "minProperties": 1,
+            "maxProperties": 1,
+            "properties": {
+                "opencensus": {
```

Comment:
This was the only part I wasn't entirely sure about.  Since this is a pluggable interface, i've made them objects, with additional properties allowed.  I think that would allow adding configuration to these readers in the future.

Reply:
As mentioned in another comment, we only add schema surface area for components which are explicitly definitely in the spec. Opencensus meets this critiera: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/compatibility/opencensus.md#metrics--stats

Reply:
OpenCensus is now the only one in this list.